pkg/client/apiutil/apimachinery.go

```diff
@@ -93,5 +102,8 @@ func createRestConfig(gvk schema.GroupVersionKind, baseConfig *rest.Config) *res
 	if cfg.UserAgent == "" {
 		cfg.UserAgent = rest.DefaultKubernetesUserAgent()
 	}
+	if cfg.ContentType == "" && localScheme.Recognizes(gvk) {
+		cfg.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
Nit: Can you use `runtime.ContentTypeProtobuf` here?

Reply:
Done